webapp/src/components/toggleSwitch/index.tsx

```diff
@@ -15,10 +14,10 @@ const ToggleSwitch = ({
     active,
     onChange,
     label,
-    labelPositioning = ToggleSwitchLabelPositioning.Left,
```

Comment:
Why this change?

Reply:
we should use type, not enum as using `type` is easier and bounds the values

Reply:
Here we're providing value using `enum/constant` instead of hard-coded values. Shouldn't we use `enum`/`constants` in such cases?

Reply:
we can do that but we should take leverage of TS wherever we can use type and literals we can get rid of import/exports

Reply:
Yeah, I understand that. But for providing values, we can't use `type`, right? For that, we'll either have to use `enum` or `constant`, right?

Reply:
@sooraj-shukla let's sync on this I think there is come confusion